### PR TITLE
Error message content changes

### PR DESCRIPTION
**Pull request by AimeeHeywood**
In response to the internal accessibility review, I have made the error messages more descriptive.

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -115,14 +115,14 @@ whichBestDescribesYou.heading = Who is this application for?
 whichBestDescribesYou.businessOwner = The business or organisation that I work for or own
 whichBestDescribesYou.businessRepresentative = A different business or organisation that I am representing
 whichBestDescribesYou.checkYourAnswersLabel = This application is for
-whichBestDescribesYou.error.required = Select who this application is for
+whichBestDescribesYou.error.required = Select if this application is for the business or organisation you work for or own or a business, organisation or individual that you're representing
```
Comment (gokyo) [inline conf/messages.en]: if you want to keep this long sentence, I suggest to add more commas to it

Comment (pllaurado) [inline conf/messages.en]: wrong character ' . replace it for **you’re**

File: conf/messages.en [hunk 2]
```diff
@@ -135,8 +135,8 @@ registerBusinessRepresenting.town = Town or city
 registerBusinessRepresenting.postCode = Postcode
 registerBusinessRepresenting.country = Country
 registerBusinessRepresenting.checkYourAnswersLabel =  Their registered details
-registerBusinessRepresenting.error.eoriNumber.required = Enter their EORI number
-registerBusinessRepresenting.error.eoriNumber.format = Enter their EORI number in the correct format
+registerBusinessRepresenting.error.eoriNumber.required = Enter the EORI number of the business, organisation or individual that you're representing
```
Comment (pllaurado) [inline conf/messages.en]: wrong character ' . replace it for you’re

Comment (SteveSugden) [reply conf/messages.en]: Needs to be a smart apostrophe e.g. you’ll .  (Alt=Shift-} on Mac keyboard)

File: conf/messages.en (hunk 1 above)
Comment (SteveSugden) [reply conf/messages.en]: Needs to be a smart apostrophe e.g. you’ll .  (Alt=Shift-} on Mac keyboard)

**Comment by SteveSugden** [comment]
DO_NOT_MERGE.

Some of these changes are going to conflict with the changes to confidential information that I'm currently doing.

So I'm going to merge this branch onto mine.  (And re-run the unit and ATs)

**Pull request closed without merging**